src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -27943,13 +27954,31 @@ public class C
     [DisallowNull] string? P1 { get; set; } = """";
     [AllowNull] string P2 { get; set; } = """";
     [MaybeNull] string P3 { get; set; } = """";
-    [NotNull] string? P4 { get; set; } = """";
+    [NotNull] string? P4 { get; set; } = """"; // 1
+
+    [DisallowNull] string? P5 { get; set; } = null; // 2
+    [AllowNull] string P6 { get; set; } = null;
```

Comment:
😕 I would expect a warning for `P6`, because this is essentially the same as `P3` in test `AllowNull_Property_WithNotNull` (where a warning is reported)